js/main.js

```diff
@@ -19,9 +20,104 @@ function formSubmit() {
     photoUrl: photoInput.value,
     text: notes.value,
     entryID: nextEntryId
-  }; nextEntryId += 1;
-  entres.push(newObj);
+  };
+  nextEntryId += 1;
+  data.entries.unshift(newObj);
   oldImg.src = '/images/placeholder-image-square.jpg';
   form.reset();
+  var newElement = renderEntry(newObj);
+  ul.prepend(newElement);
+  if (data.entries.length !== 0) {
+    toggleNoEntries();
+  }
+  viewSwap('entries');
 }
 form.addEventListener('submit', formSubmit);
+
+function renderEntry(entry) {
+
+  var li = document.createElement('li');
+
+  var div1 = document.createElement('div');
+  div1.setAttribute('class', 'row');
+  li.appendChild(div1);
+
+  var img = document.createElement('img');
+  img.src = entry.photoUrl;
+  img.setAttribute('class', 'column-half');
+  div1.appendChild(img);
+
+  var div2 = document.createElement('div');
+  div2.setAttribute('class', 'column-half');
+  div1.appendChild(div2);
+
+  var h2 = document.createElement('h2');
+  h2.textContent = entry.title;
+  div2.appendChild(h2);
+
+  var p = document.createElement('p');
+  p.textContent = entry.text;
+  div2.appendChild(p);
+
+  return li;
+}
+
+function domCreation() {
+  var ul = document.querySelector('ul');
+  for (var i = 0; i < (data.entries).length; i++) {
+    var newElement = renderEntry(data.entries[i]);
+    ul.appendChild(newElement);
+  }
+
+  data.view = localStorage.getItem('current screen');
```

Comment:
You shouldn't be grabbing from localStorage here, you should just be referencing `data.view`. This will work once you save the whole data object to localStorage